docs/reference.rst

```diff
@@ -114,6 +114,101 @@ Client
         main();
 
 
+.. js:function:: createHttpClient(options?: string | ConnectOptions | null): Client
+
+    Creates an HTTP client for interacting with an EdgeDB instance from a browser or edge runtime environment. This function returns an instance of :js:class:`Client` configured to communicate over HTTP using the Fetch API.
+
+    :param options:
+        This is an optional parameter. When it is not specified the client
+        will connect to the current EdgeDB Project instance.
+
+        If this parameter is a string it can represent either a
```

Comment:
The newlines here get collapsed. In the rendered output, "If" is directly against the preceding period with nothing between. I've started a thread about it on Slack. Seems like a bug in the docs.